core/metadata/migrations.go

```diff
@@ -239,13 +239,13 @@ func migrateSandboxes(tx *bolt.Tx) error {
 			})
 		})
 	}); merr != nil {
-		return fmt.Errorf("failed to copy sandboxes into v1 bucket: %w", err)
+		return fmt.Errorf("failed to copy sandboxes into v1 bucket: %w", merr)
 	}
 
 	for _, ns := range deletingBuckets {
 		derr := tx.DeleteBucket(ns)
```

Comment:
if we change; probably would be good to also inline this err in the if-statement (`if err := tx.DeleteBucket(ns); err != nil {`). lts slightly more idiomatic, and clearly indicates that it's a local var.